### DISCUSSION TITLE
Forgotten set for URL

### DISCUSSION THREAD

**Discussion by mdehganr** [Dotnet-Boxed/Templates#185] [Q&A]
Dear Sir

I think there is a little bug on definition of URL in your class, like url is forgetten to be writable, please check attachment and fix it if it is bug.

Regards
![image](https://user-images.githubusercontent.com/16499516/35185250-5f243ae4-fe16-11e7-9707-85b47707ae32.png)

**Comment by RehanSaeed** [comment]
I don't understand what you mean, could you explain further?

**Comment by RehanSaeed** [comment]
Feel free to reopen if you would like to elaborate.
